src/svcs/_core.py

```diff
@@ -194,6 +194,10 @@ def __del__(self) -> None:
                 stacklevel=1,
             )
 
+    def __iter__(self) -> Iterator[type]:
```

Comment:
I would appreciate an input - whether adding this method is OK and if so, where / how should it get documented

Reply:
Since this looks like #66, it's worth adding, but it might better as a separate PR for the reason alone that I'm not sure how long this one will take and it seems less controversial/decision-heavy.

Reply:
Could this method only be included in another PR? Or should registered services inspection be more extensive?

Reply:
Yes, adding `__iter__` in a separate PR is what I meant. Two changelog entries are a strong indicator that a PR is doing too much. 🤓

I also do think that the inspection could use more knobs, but this would be a start.